src/main/java/org/mqttbee/api/mqtt/MqttClientData.java

```diff
@@ -35,7 +35,13 @@
 
     int getServerPort();
 
-    boolean usesSSL();
+    @NotNull
+    Optional<MqttWebsocketConfig> getWebsocketConfig();
+
+    boolean usesSsl();
```

Comment:
I would like the ordering of the methods to be:
usesSsl
getSslConfig
usesWebsockets
getWebsocketConfig
getExecutorConfig
...

Reply:
DONE